sdk/communication/communication-chat/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/communication-chat",
-  "version": "1.3.2",
+  "version": "1.4.0",
```

Comment:
Need to update "@azure/communication-signaling" version

Reply:
no beta for this one?

Reply:
https://www.npmjs.com/package/@azure/communication-chat/v/1.4.0-beta.2